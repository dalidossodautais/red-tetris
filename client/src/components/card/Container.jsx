import styled from "styled-components";

export default styled.div`
  background-color: #423434;
  border-radius: 5px;
  box-shadow: -10px 10px 15px #2e2424;
  height: 100%;
  margin-right: 10px;
  width: 500px;
`;
