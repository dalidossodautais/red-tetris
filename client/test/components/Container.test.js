import React from "react";
import renderer from "react-test-renderer";
import Container from "../../src/components/Container";

describe("Container", () => {
  test("Standard", () => {
    const component = renderer.create(<Container />);
    // rendering
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
    // re-rendering
    tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
