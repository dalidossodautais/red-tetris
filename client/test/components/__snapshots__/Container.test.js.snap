// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container Standard 1`] = `
<div
  className="sc-bdfBwQ jSaWsD"
/>
`;

exports[`Container Standard 2`] = `
<div
  className="sc-bdfBwQ jSaWsD"
/>
`;
